### ISSUE TITLE
How to format attributes

### ISSUE THREAD

**Issue by momala454** [squizlabs/PHP_CodeSniffer#3656]
**Describe the bug**
Let's say I have this

```php
#[OA\Post(path: '/users/register', description: 'Create an user account')]
```

How do I make it like this

```php
#[OA\Post(
    path: '/users/register',
    description: 'Create an user account'
)]
```
PSR2.Methods.FunctionCallSignature or PEAR.Functions.FunctionCallSignature with `allowMultipleArguments` to false doesn't seem to work

**Versions (please complete the following information):**
 - OS: windows 10
 - PHP: 8.1
 - PHPCS: 3.7.1
 - Standard: psr12

**Comment by remy-theroux** [comment]
I have searched a rule to allow multi attributes but have not found yet.
FunctionCallSignature should avoid analyse attributes.

**Comment by remy-theroux** [comment]
I will have a look thx

**Comment by rodrigoprimo** [comment]
A similar request has been filed in the new PHP_CodeSniffer repository: PHPCSStandards/PHP_CodeSniffer#1202. Linking for visibility.
